galaxykit/command.py

```diff
@@ -283,10 +283,30 @@ def main():
                     if not args.ignore:
                         print(e)
                         sys.exit(EXIT_NOT_FOUND)
+
+            elif args.operation == "delete-in-repository":
```

Comment:
Just a heads-up that when the CLI refactor that @ShaiahWren and I have been working on gets merged in this functionality will probably get exposed under an optional `--repository` flag that has `published` as the default value.

Reply:
Sounds good :-) How is it going with this refactor?

Reply:
It's nearly done I think, I just need to sit down and hammer out the final bits. Lots of competing priorities for my attention at the moment unfortunately. I'd say we could expect it to be done in the next two weeks, if nothing else comes onto my plate.

Reply:
Ok, I will wait :-)